src/css/_nav.scss

```diff
@@ -0,0 +1,39 @@
+.nav-btn {
+  align-items: center;
+  background: #5B7894;
+  border: none;
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  font-size: 0px;
+  font-weight: 700;
+  justify-content: center;
+  margin: 0%;
+  outline: none;
+  overflow: hidden;
+  text-align: center;
+  transition: .4s ease;
+  width: 15%;
+}
+
+.nav-btn:hover {
+    color: $primary-green;
+    font-size: 14px;
+    text-shadow: 0px 2px 4px black;
+    transition: .4s ease;
+}
+
+.nav-btn img {
```

Comment:
We may be able to nest these (`.nav-btn:hover` and `.nav-btn img`) inside `.nav-btn`

Reply:
I tried nesting the hover in the nav but for some reason the My Recipes and the My Pantry animations wouldn't show up when hovering over their icons.  We can look into this tomorrow. I would like to nest them if possible.

Reply:
Looks great y'all!